app/controllers/api/v1/downvotes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::DownvotesController < ApplicationController
+  def create
+    downvote = Downvote.new(user: current_user)
+    downvote.review = Review.find(params[:review_id])
+    reviews = Review.where(videogame: downvote.review.videogame)
+    if downvote.save 
+      if current_user.upvotes.find_by(review_id: params["review_id"]) != nil
+        current_user.upvotes.find_by(review_id: params["review_id"]).destroy
+      end
+      render json: reviews
+    else
+      render json: { errors: downvote.errors.full_messages.to_sentence }
+    end
+  end
+end
```

Comment:
haha, while this implementation might have been simpler with one Vote model (fewer database transactions in the long term) this is a clean and rather CRUDy (in a good way) implementation. 

`downvote.review.videogame` could probably be dried up with a custom association / method, but is largely a ok